guides/user/managed-delivery/CI-features/index.md

```diff
@@ -0,0 +1,66 @@
+---
+layout: single
+title:  "CI features"
+sidebar:
+  nav: guides
+---
+
+{% include toc %}
+
+One of the goals of Managed Delivery is to show you the journey of your code, from commit to deployment,
+in an easy, consistent way.
+In order to enable this experience, we created an integration between Managed Delivery and the Continuous Integration (CI) provider, which is described below.
+
+### Note: in order to get it working for you, you'll have to integrate your CI provider. Please see our guide on [CI integration](/guides/developer/extending/integrate-your-CI).
```

Comment:
Feel free to change this, I'm just trying to remove any suggestion that the end user would be the one implementing a new CI provider for Spinnaker as this implied originally.
```suggestion
### Note: in order to take advantage of this feature, your Spinnaker operator/administrator will have to configure a CI provider which matches your company's setup. They can look for available open-source or commercial CI provider implementations, or even develop their own (see our developer guide on [CI integration](/guides/developer/extending/integrate-your-CI)).
```